server/s3_object_metadata.cc

```diff
@@ -406,6 +406,13 @@ void S3ObjectMetadata::set_part_index_layout(
   motr_part_layout_str = S3M0Uint128Helper::to_string(part_index_layout);
 }
 
+void S3ObjectMetadata::set_null_object_version_id(std::string null_version_id) {
```

Comment:
this can be (const std::string&  null_version_id),,,,,

Reply:
Done